.verb.md

```diff
@@ -0,0 +1,11 @@
+{%= badge("npm") %} {%= badge("standard") %}
+
+# {%= name %} {%= badge("travis") %}
+
+## API
+
+{%= apidocs("index.js") %}
```

Comment:
This [does not yield the expected output](https://github.com/assemble/verb/issues/48) so I [manually put in a link](https://github.com/PolicyStat/call-n-times/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8R8).